adapters/ion-adapter/src/main/java/com/connexta/replication/adapters/ion/IonNodeAdapter.java

```diff
@@ -119,6 +119,7 @@ public boolean createResource(CreateStorageRequest createStorageRequest) {
     builder.part("file", getResourceEntity(resource));
     builder.part("metacard", getMetadataEntity(resource.getMetadata()));
     builder.part("correlationId", resource.getId());
+    builder.part("metadata.modified", resource.getMetadata().getMetadataModified().toString());
```

Comment:
✏️ Let's not call this metadata as it will get problematic in the long run when a piece of intel can have multiple different type of metadata. This is the modified timestamp for the intel from the perspective of Ion.

What is the expected format? I doubt we want to rely on the `Date.toString()` method to fill this.

The product and all the metadata types could have their own modified timestamp. 

❓ Should Ion store one for each? As such, we would send one for the intel and one for the metacard passed in this message.

We need to figure out which one we associate with the intel from the DDF's perspective such that it is enough to distinguish different versions.

Reply:
Isn't each metadata type just another form on the same metadata? In that case each one should have the same metadata modified time.

I think Date.toString() uses ISO format and can easily be parsed by other java applications so I'm not sure if there's anything wrong with it. This should just be used by cdr I think and does need to be to some I2O specification.

Reply:
metadata can be updated/created at different time so we might want to keep track of each individually in the future. For this PR, we care about the intel and should reference the modified timestamp for the intel along with its correlation id already provided.

As for the time format, it is up to the ingest API to dictate what they want to support. That being said, the OpenAPI spec indicates the format should follow RFC3339 (https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#data-types) which is ISO 8601.